### PR TITLE
Feat: singnup page added

### PR DESCRIPTION
**Pull request by anurag-kkumarr**
Added signup Authentication

- Please review this PR

### CODE REVIEW

**Review by rahul31agrawal — APPROVED** [review]
1) for search bar, the search icon should be on the left side & put input & icon in the same div & give same background color (White) to both the items, It will look neat.
2) In the nav bar all the buttons like home, product, text should be at the center of the box.
3) for input like email, name you should give border radius of 0.5rem, it will look good.
Rest looks good to me, you can merge for now but please fix these things later

File: components/signup/signup.html [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/components/signup/signup.css">
+    <title>Signup</title>
+</head>
+<body>
+    <nav class="nav-bar">
+        <header class="header">WATCHKART</header>
+        <div class="menu-bar">
+            <a class="link-btn login-btn" href="/index.html">Home</a>
+            <a class="link-btn login-btn prod-btn" href="/components/products/men-prod.html">Product</a>
```
Comment (sashaboi) [inline components/signup/signup.html:19]: adding display flex, justify center and align center might alight text to center of buttons.

File: components/signup/signup.html [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        </div>
+    </nav>
+    <main class="login-container">
+
+        <!-- sign-up -->
+        <header class="login-header">Sign Up</header>
+
+        <form action="">
+            <div class="sign-layout">
+                    <div class="name-info">Enter First name</div>
```
Comment (sashaboi) [inline components/signup/signup.html:44]: There seems to be some sort of padding applied to this div making it not aligned with the other labels hope that it fixed.

File: components/signup/signup.html [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+                    <div class="pwd-info">Enter Your password</div>
+                    <input class="input-pwd" type="password" placeholder="asdfg@123" name="password" id="password">
+                    <div class="pwd-info-cnf">confirm Your password</div>
+                    <input class="input-pwd" type="password" placeholder="asdfg@123" name="password" id="password">
+                    <label for="" class="terms-cond">
+                    <input type="checkbox" name="remember" id="remember"> I agree to terms and condition.
+                </label></br>
+                <button class="log-in-btn">Register</button>
+                <div class="new-acct">Already have an account?</div>
+                <a class="log-in-btn" href="/components/login-page/login.html">login</button>
```
Comment (sashaboi) [inline components/signup/signup.html:59]: The button text needs center aligning as well.

**Review by ShoaibTamboli — COMMENTED** [review]
i would recommend you to add some padding between all text box and labels.

2) can reduce the height for search bar .
 
 Remove <a> from below line of code, 

image
image

**Review by sashaboi — COMMENTED** [review]
There seem to be two buttons on the left side empty, hope they are addressed. 
image

**Comment by anurag-kkumarr** [comment]
> 1. for search bar, the search icon should be on the left side & put input & icon in the same div & give same background color (White) to both the items, It will look neat.
> 2. In the nav bar all the buttons like home, product, text should be at the center of the box.
> 3. for input like email, name you should give border radius of 0.5rem, it will look good.
>    Rest looks good to me, you can merge for now but please fix these things later

I will make all these changes in final Refactor. Else lot of merge conflicts occur.

**Comment by anurag-kkumarr** [comment]
> There seem to be two buttons on the left side empty, hope they are addressed. image

I didn't see any button or any other element there

**Pull request merged**